docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   zoo1:
-    image: zookeeper:3.4.9
+    image: zookeeper:3.4.13
```

Comment:
This is the version of Zookeeper that Confluent Kafka v5.0.0 pulls into the service.